app/charts/map/map.tsx

```diff
@@ -231,7 +231,7 @@ export const MapComponent = () => {
               getLineColor={[255, 255, 255]}
             />
           </>
-        )}
+        )} */}
```

Comment:
Should this be removed ?

Reply:
Generally yes, I left it for the moment because maybe it also makes sense to draw the "mesh" of the shapes, but maybe a regular border would be enough – honestly I didn't think about this yet

Reply:
Just a quick note: Drawing the mesh is better because it's just 1 line for shared borders. If each shape draws its outline, this can look a bit messy.